src/intrinsics/prepack/global.js

```diff
@@ -398,6 +398,25 @@ export default function(realm: Realm): void {
     })
   );
 
+  // Helper function for Prepack developers inspect a value
+  // when interpreting a particular node in the AST.
+  global.$DefineOwnProperty(
+    "__debugValue",
+    new PropertyDescriptor({
+      value: createNativeFunctionForResidualInjection(
+        "__debugValue",
+        ([v, s]): void => {
+          debugger; // eslint-disable-line no-debugger
+        },
+        createOperationDescriptor("NOOP"),
```

Comment:
I would prefer this to be "EMPTY_STATEMENT".

Reply:
It's not outputting anything and statements can't be empty. Maybe "EMPTY_OP"

Reply:
See line 136 in /ResidualOperationSerializer.js

Reply:
I know what you mean (you're relating it to Babel's emptyStatement), but I meant as a generic operation rather than as the output JS AST node. I don't mind either way, it's all bikeshedding really anyway.

Reply:
I agree that it is bikeshedding. However, now that you brought it up, I prefer "NOOP." "EMPTY_STATEMENT" ties in with other constants used *lexically*. But try a google search for "NOOP" and then one for "EMPTY_STATEMENT" and decide which one contains more visual signal.